### PR TITLE
Update to babel 6

### PR DESCRIPTION
**Pull request by aardmark**
Hi There,

This PR includes updates for Babel -> 6, eslint -> 3 and istanbul to 1. Thought you might be able to use it.

Regards,
Mark

### CODE REVIEW

File: generators/gulp/templates/babelrc [hunk 1]
```diff
@@ -1 +1,3 @@
-{}
+{
+    "presets": [ "es2015" ]
```
Comment (SBoudrias) [inline generators/gulp/templates/babelrc]: Remove the inline spaces in the array

File: generators/gulp/templates/gulpfile.js [hunk 2]
```diff
@@ -88,7 +88,7 @@ gulp.task('line-ending-corrector', function () {
 
 gulp.task('babel', ['clean'], function () {
   return gulp.src('<%- projectRoot %>')
-    .pipe(babel())
+    .pipe(babel({presets: ['es2015']}))
```
Comment (SBoudrias) [inline generators/gulp/templates/gulpfile.js]: Can this use the `.babelrc` file so settings are not duplicated?

**Comment by aardmark** [comment]
I've made both those changes. The settings in the call to gulp-babel were redundant.

**Comment by SBoudrias** [comment]
Awesome! thanks :D

**Pull request merged**